conf/pacific/rgw/ceph_cluster+standalone_osp_node.yaml

```diff
@@ -0,0 +1,47 @@
+globals:
+  - ceph-cluster:
+      name: ceph
+      node1:
+        disk-size: 20
+        no-of-volumes: 4
+        role:
+          - _admin
+          - installer
+          - mgr
+          - mon
+          - osd
+      node2:
+        disk-size: 20
+        no-of-volumes: 4
+        role:
+          - mgr
+          - mon
+          - osd
+      node3:
+        disk-size: 20
+        no-of-volumes: 4
+        role:
+          - mon
+          - osd
+      node4:
+        disk-size: 20
+        no-of-volumes: 4
+        role:
+          - osd
+      node5:
+        role:
+          - rgw
+      node6:
+        role:
+          - client
+  - ceph-cluster:
```

Comment:
would it be nice, if we keep OSP conf separately and use with any configuration in future.